omf-python/src/geometry.rs

```diff
@@ -42,13 +44,30 @@ pub struct PyPointSet {
 
 #[pymethods]
 impl PyPointSet {
-    fn get_origin(&self) -> [f64; 3] {
-        //  basic types don't need .clone()
+    #[getter]
+    fn origin(&self) -> [f64; 3] {
         self.inner.origin
     }
 
-    // #[getter]
-    // fn get_vertices(&self) -> Vec<Array<Vertex>> {
-    //     self.inner.vertices
-    // }
+    fn get_vertices(&self, reader: &PyReader) -> PyResult<Vec<[f64; 3]>> {
+        let vertices = reader
+            .inner
+            .array_vertices(&self.inner.vertices)
+            .map_err(|e| PyErr::new::<PyIOError, _>(e.to_string()))?;
+        vertices
+            .map(|result| result.map(|[x, y, z]| [x, y, z]))
```

Comment:
This looks a bit odd that its mapping them to the same thing? Or does this do something else?

Reply:
This part is likely to change, I was previously trying to cast them to f64 because it looks like array_vertices can return f32 or f64, I'm not sure if we should/can handle the different return types or if I'm misreading the array_vertices function and it does the type casting for us. But you're right, it does look odd :-)

Reply:
I've moved the array_vertices function to the Reader object and tidied up the code a bit- if we hit a problem between f32/f64 vertices we'll tackle it when we see it. I've left it just accepting a `PyArrayVertex` so it can work with AttributeData too